templates/homepage.html

```diff
@@ -1,95 +1,15 @@
-{% load static %}
-
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
-    <link
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
-      rel="stylesheet"
-      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
-      crossorigin="anonymous"
-    />
-    <title>Falcon</title>
-    <link rel="icon" type="image/x-icon" href="/static/img/favicon/favicon.ico">
-    <style>
-      body {
-        background:linear-gradient(0deg, rgba(204, 204, 204, 0.9), rgba(255, 255, 255, 0.9)), url('/static/img/homepage_bg.webp');
-        background-repeat: no-repeat;
-        background-attachment: fixed;
-        background-size: cover;
-
-      }
-    </style>
-  </head>
-  <body id="page-top" class="mt-3">
-    <div style="height: 100px"></div>
-    <!-- Navigation-->
-    <nav
-      class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top"
-      id="mainNav"
-    >
-      <div class="container px-4">        
-        <a class="navbar-brand" href="/">
-          <img src="{%  static   'img/favicon/favicon-32x32.png' %}" width="30" height="30">
-          Falcon
-        </a>
-        <button
-          class="navbar-toggler"
-          type="button"
-          data-bs-toggle="collapse"
-          data-bs-target="#navbarResponsive"
-          aria-controls="navbarResponsive"
-          aria-expanded="false"
-          aria-label="Toggle navigation"
-        >
-          <span class="navbar-toggler-icon"></span>
-        </button>
-        <div class="collapse navbar-collapse" id="navbarResponsive">
-          <ul class="navbar-nav ms-auto">
-            <li class="nav-item">
-              <a class="nav-link" href="/login">Login</a>
-            </li>
-          </ul>
+{% extends "base.html" %}
+{% block content %}
+    {% for task in tasks %}
+       <article class="media content-section">
+        <div class="media_body">
+            <div class="article_metadata">
+                <a class="mr-2" href="#">{{ task.assigneeTasks }}</a>
+                <small class="text-muted">{{ task.created_date|date:"F d, Y" }}</small>
+            </div>
+            <h3><a class="article_title" href="#">{{ task.title }}</a></h3>
+            <p class="article-content">{{ task.description }}</p>
```

Comment:
As I mentioned above, the tasks shouldn't be displayed here

Reply:
Talk to me in private I explained ... it gives you that you enter the active tasks

Reply:
Or is right

Reply:
I explained... can sit down to check the second option and decide..) I see no logical reason to make another page when the home page currently has no purpose...